cli/tests/integration/test_tests.rs

```diff
@@ -180,11 +180,12 @@ itest!(ops_sanitizer_multiple_timeout_tests_no_trace {
   output: "test/ops_sanitizer_multiple_timeout_tests_no_trace.out",
 });
 
-itest!(ops_sanitizer_missing_details {
-  args: "test --allow-write --allow-read test/ops_sanitizer_missing_details.ts",
-  exit_code: 1,
-  output: "test/ops_sanitizer_missing_details.out",
-});
+// TODO(@littledivy): re-enable this test, recent optimizations made output non deterministic.
+// itest!(ops_sanitizer_missing_details {
+//  args: "test --allow-write --allow-read test/ops_sanitizer_missing_details.ts",
+//  exit_code: 1,
+//  output: "test/ops_sanitizer_missing_details.out",
+// });
```

Comment:
This test depends on the `read` op to take long enough so that the op sanitizer detects it before completion...but the optimizations reading faster (lol). Disabled it for this PR. cc @bartlomieju

Reply:
That's fine, just add a link to https://github.com/denoland/deno/issues/14268